userguide/tutorials/payment_plugin.adoc

```diff
@@ -1,61 +1,575 @@
+= Payment Plugin Overview
+
+Payment plugins integrate Kill Bill with a specific gateway (i.e. payment processor), such as Stripe or Braintree. We already have many open-source payment plugins available on GitHub as follows:
+
+. https://github.com/killbill/killbill-stripe-plugin[killbill-stripe-plugin] - Plugin to use  https://stripe.com/[Stripe] as the payment gateway
+
+. https://github.com/killbill/killbill-adyen-plugin[killbill-adyen-plugin] - Plugin to use https://www.adyen.com/[Adyen] as the payment gateway
+
+. https://github.com/killbill/killbill-paypal-express-plugin[killbill-paypal-express-plugin] - Plugin to use https://www.paypal.com/webapps/mpp/express-checkout[PayPal Express Checkout] as the payment gateway
```

Comment:
Let's remove that one as it's an old version. A new one is about to be released (the link isn't public yet though unfortunately).

Reply:
@pierre , Does this refer to PayPal Express Checkout or Adyen or both?

Reply:
Just PayPal Express Checkout (and Litle below) -- you can keep Adyen.